addons/website/views/snippets/s_image_gallery.xml

```diff
@@ -12,11 +20,11 @@
 <template id="s_image_gallery_options" inherit_id="website.snippet_options">
     <xpath expr="." position="inside">
         <div data-js="gallery" data-selector=".o_gallery">
-            <we-select string="Mode">
+            <we-select string="Mode" data-dependencies="!slideshow_mode_opt">
                 <we-button data-mode="grid" data-name="grid_mode_opt">Grid</we-button>
                 <we-button data-mode="masonry" data-name="masonry_mode_opt">Masonry</we-button>
                 <we-button data-mode="nomode">Float</we-button>
-                <we-button data-mode="slideshow" data-name="slideshow_mode_opt">Slideshow</we-button>
+                <we-button data-mode="slideshow" data-name="slideshow_mode_opt" class="d-none">Slideshow</we-button>
```

Comment:
What about removing the last option instead of hiding it and adapt the data-dependencies to say "if option 1 ou 2 ou 3" with a comment to explain?

Also I see that the code to be able to switch to slideshow is still there, is this not possible to remove that and clean the code a minimum?

Reply:
For point 2 , you mean use a select-class instead of the custom method "data-mode" ?

Reply:
No, I said that the JS code for switching to slideshow mode is still there and should be removed

Reply:
Point 1: I actually need the "slideshow_mode_opt" to display slideshow options (Indicators style etc.)
Point 2: If you mean the "mode" method, we still need it even for slideshow because after adding images we use this method with the result of getMode() in parameter. But I may have misunderstood...

Reply:
Ok, let's leave like that then :+1: